base/CMakeLists.txt

```diff
@@ -51,6 +52,33 @@ therock_cmake_subproject_provide_package(rocm-core rocm-core lib/cmake/rocm-core
 therock_cmake_subproject_activate(rocm-core)
 
 
+################################################################################
+# amdsmi
+################################################################################
+
+if(NOT WIN32)  # TODO(#36): Enable on Windows and/or make subproject inclusion generally optional
+
+therock_cmake_subproject_declare(amdsmi
+  EXTERNAL_SOURCE_DIR "amdsmi"
+  USE_DIST_AMDGPU_TARGETS
+  BACKGROUND_BUILD
+  CMAKE_ARGS
+    # TODO: Enable tests
+    # -DBUILD_TESTS=${THEROCK_BUILD_TESTING}
+  BUILD_DEPS
+    therock-googletest
+  RUNTIME_DEPS
+    rocm-core
+)
+therock_cmake_subproject_provide_package(amdsmi
+  amd_smi lib/cmake)
+# therock_cmake_subproject_provide_package(amdsmi
+#   ROCM share/rocm/amdsmi)
+therock_cmake_subproject_activate(amdsmi)
```

Comment:
Is the commented out `therock_cmake_subproject_provide_package` going to be used later, or can that be removed?